shoper.py

```diff
@@ -0,0 +1,135 @@
+import os
+import pandas as pd
+from operator import itemgetter
+from re import search
+
+
+def deactivate_products(database, dc_price, replacement_price):
+    """
+    Deactivates products set with exact price - e.g. 107.99
```

Comment:
Każdy produkt z ceną x zostanie zablokowany. Czy tak ma być? Dlaczego?
Co jak to będą inne produkty z tą samą ceną?